charts/lakefs/templates/job.yaml

```
@@ -13,6 +13,11 @@ spec:
   template:
     metadata:
       name: {{ include "lakefs.fullname" . }}-migrate
+      annotations:
+      {{- with .Values.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

from what I read online there is a shorter version :)

```suggestion
      {{ toYaml .Values.podAnnotations | indent 8 }}
```